server/SeparateStainsMacenkoPCA/SeparateStainsMacenkoPCA.xml

```diff
@@ -30,7 +30,6 @@
       <longflag>samplePercent</longflag>
       <description>Percentage of pixels to sample.  Specify either this or --sampleApproximateTotal</description>
       <constraints>
-	<minimum>0</minimum>
 	<maximum>1</maximum>
```

Comment:
Yes, I think it would be better to change the `_percent` to `_fraction` where appropriate.